### ISSUE TITLE
Add "Handle_4_Directions" to the Floor Observation Room

### ISSUE THREAD

**Issue by LautaroL20** [TeamSpen210/BEE2-items#932]
![palette](https://cloud.githubusercontent.com/assets/12244376/19418283/27a977b8-9397-11e6-85b3-81311206b86b.png)

**Comment by IceTudor** [comment]
Why is this item even here?
What kind of testing facility has windows on the floors, in an enclosed testing area?

**Comment by ZombiePwner1** [comment]
@IceTudor You never know some portal mods may have observation rooms on the floor + ceiling.

**Comment by Endermage77** [comment]
@IceTudor: Floor lights. Lights in the floor. For lighting things.
The observation rooms are not always necessarily actual observation rooms.

**Comment by TeamSpen210** [comment]
I'd want to make it ignored on walls.

**Comment by Endermage77** [comment]
Who knows?
Maybe gravity in the testing chamber is different.

"In this next test, we attached a gravity generator to the testing environment to see if you can test while upside-down, so don't freak out if all the windows are upside-down."

**Comment by TeamSpen210** [comment]
No.

**Comment by Luke18033** [comment]
I'm not really sure what the point of this is?

**Comment by LautaroL20** [comment]
Basically, allow the posibility to being able to choose in which position the floor observation room will be builded so players can see them in different positions from the floor surface.

**Comment by Luke18033** [comment]
Yes, but...how is that useful? Floor/ceiling observation rooms are intentionally built without a "forward" direction.

**Comment by Luke18033** [comment]
If it's just to add more variation, that could be done with random rotation.

**Comment by LautaroL20** [comment]
Random rotation would be fine.
